Dockerfile

```diff
@@ -1,74 +1,142 @@
+    mv /usr/src/osmborder/build/src/osmborder_filter /build-bin
+
+
+# Primary image
+FROM python:3.8-slim
+LABEL maintainer="Yuri Astrakhan <YuriAstrakhan@gmail.com>"
+
+ARG PG_MAJOR=12
+ARG VT_UTIL_VERSION=v2.0.0
+ARG TOOLS_DIR=/usr/src/app
+
+WORKDIR ${TOOLS_DIR}
 
 # Using VT_UTIL_DIR and OMT_UTIL_DIR vars allow users to provide custom util files:
 # postgis-vt-util.sql and language.sql
 # See README
-ENV PATH="/usr/src/app:${PATH}" \
-    VT_UTIL_DIR=/opt/postgis-vt-util \
-    OMT_UTIL_DIR=/usr/src/app/sql \
+ENV VT_UTIL_DIR=/opt/postgis-vt-util \
+    OMT_UTIL_DIR="${TOOLS_DIR}/sql" \
+    TOOLS_DIR="$TOOLS_DIR" \
     SQL_DIR=/sql \
-    OSMBORDER_REV=e3ae8f7a2dcdcd6dc80abab4679cb5edb7dc6fa5 \
-    PGFUTTER_VERSION="v1.2" \
-    WGET="wget --quiet --progress=bar:force:noscroll --show-progress"
+    PATH="${TOOLS_DIR}:${PATH}" \
+    IMPORT_DIR=/import \
+    IMPOSM_CACHE_DIR=/cache \
+    MAPPING_YAML=/mapping/mapping.yaml \
+    DIFF_DIR=/import \
+    TILES_DIR=/import \
+    CONFIG_JSON=${TOOLS_DIR}/config/repl_config.json
+
 
-ARG PG_MAJOR=12
 
-RUN curl https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add - \
-    && /bin/bash -c 'source /etc/os-release && echo "deb http://apt.postgresql.org/pub/repos/apt/ ${VERSION_CODENAME?}-pgdg main ${PG_MAJOR?}" > /etc/apt/sources.list.d/pgdg.list' \
-    && DEBIAN_FRONTEND=noninteractive apt-get update \
-    && DEBIAN_FRONTEND=noninteractive apt-get install  -y --no-install-recommends \
+RUN set -eux ;\
+    /bin/bash -c 'echo ""; echo ""; echo "##### Installing packages..."' >&2 ;\
+    DEBIAN_FRONTEND=noninteractive apt-get update ;\
+    DEBIAN_FRONTEND=noninteractive apt-get install  -y --no-install-recommends \
+        # a few common tools
+        ca-certificates \
+        curl \
+        wget \
+        git  \
+        gnupg2  `# TODO: not sure why gnupg2 is needed`  ;\
+    curl https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add - ;\
+    /bin/bash -c 'source /etc/os-release && echo "deb http://apt.postgresql.org/pub/repos/apt/ ${VERSION_CODENAME:?}-pgdg main ${PG_MAJOR:?}" > /etc/apt/sources.list.d/pgdg.list' ;\
+    DEBIAN_FRONTEND=noninteractive apt-get update ;\
+    DEBIAN_FRONTEND=noninteractive apt-get install  -y --no-install-recommends \
         aria2     `# multi-stream file downloader` \
         graphviz  `# used by layer mapping graphs` \
         sqlite3   `# mbtiles file manipulations`   \
-        gdal-bin  `# installs ogr2ogr` \
+        gdal-bin  `# contains ogr2ogr` \
         osmctools `# osmconvert and other OSM tools` \
-        postgresql-client-${PG_MAJOR?} \
-        \
-        `# tools to build osmborder` \
-        build-essential \
-        ca-certificates \
-        cmake \
-        git \
-        libosmium2-dev \
-        wget \
-        zlib1g-dev \
+        osmosis   `# useful toolset - https://wiki.openstreetmap.org/wiki/Osmosis` \
+        postgresql-client-${PG_MAJOR:?}  `# psql` \
```

Comment:
Here `${VAR:?}`

Reply:
In the main and current release, it is [`${VAR?}`](https://github.com/openmaptiles/openmaptiles-tools/blob/master/Dockerfile#L27)